tests/compile_fail/try_from/invalid_repr.rs

```diff
@@ -0,0 +1,7 @@
+#[derive(derive_more::TryFrom)]
+#[repr(a + b)]
+enum Enum {
```

Comment:
@ModProg it would also be nice to cover the case with _valid_ `repr`, but not meaningful, like `repr(Rust)` or `repr(C)`.

I think, we should control the exact list of types we accept, inside the macro: `i8`, `u8`, `i16`, `u16`, `i32`, `u32`, `i64`, `u64`, `i128`, `u128`, `isize`, `usize`. And disregard anything else.